README.md

```diff
@@ -130,6 +130,47 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
         // The zone of the target iRODS server.
         "zone": "<string>",
 
+        // Defines options for secure communication with the target iRODS server.
+        "ssl": {
```

Comment:
Perhaps this should be `tls`?

Which is more correct? SSL or TLS?

Reply:
TLS is more correct/updated.

Reply:
"Transport Layer Security (TLS) is the upgraded version of SSL that fixes existing SSL vulnerabilities."

Reply:
We need to go through and update the iRODS server as well one day, but that's a bigger lift.

Out here, we should use the good/new words.

Reply:
Updated to use `tls`.